<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="methods" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="methods" name="Test(failed)">
    <classes>
      <class name="amazon_Project1.Project1_Amazon.Testcase13_applycoupen">
        <methods>
          <include name="applycoupen"/>
          <include name="launchbrowser"/>
          <include name="quit"/>
        </methods>
      </class> <!-- amazon_Project1.Project1_Amazon.Testcase13_applycoupen -->
      <class name="amazon_Project1.Project1_Amazon.TestCase10_AddandRemoveCart_Project1">
        <methods>
          <include name="addandremove"/>
          <include name="launchbrowser"/>
          <include name="quit"/>
        </methods>
      </class> <!-- amazon_Project1.Project1_Amazon.TestCase10_AddandRemoveCart_Project1 -->
      <class name="amazon_Project1.Project1_Amazon.TestCase02_Loginwithdataprovider">
        <methods>
          <include name="loginwithcorrectcred" invocation-numbers="1"/>
          <include name="launchbrowser"/>
          <include name="quit"/>
        </methods>
      </class> <!-- amazon_Project1.Project1_Amazon.TestCase02_Loginwithdataprovider -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
